EIPS/eip-4519.md

```diff
@@ -209,12 +209,12 @@ This EIP proposes using the Smart Contract to verify the mutual authentication p
 This EIP proposes the attributes timestamp (to register in the blockchain whenever the physical asset checks the tie with its token) and timeout (to register the maximum delay time established for the physical asset to prove again the tie). These attributes avoid a malicious owner or user could use the asset endlessly.
```

Comment:
Is the timestamp measured in blocks, seconds, or milliseconds?

Reply:
Would you mind specifying that?

Also, I would much prefer if `block.number` is used.

Reply:
We prefer to use `block.timestamp` since it is more difficult for an electronic device to estimate the number of blocks than the time elapsed since the last time the timestamp was updated.